docs/src/components/banner/AppFrameBanner.tsx

```diff
@@ -5,6 +5,15 @@ import FEATURE_TOGGLE from 'docs/src/featureToggle';
 import PageContext from 'docs/src/modules/components/PageContext';
 import { convertProductIdToName } from 'docs/src/modules/components/AppSearch';
 
+const showSurveyMessage = false;
+
+function isBlackFriday() {
+  const today = Date.now();
+  const start = new Date('2024-11-25').getTime();
```

Comment:
I merged this too quickly, it will create an hydration error and a CLS after the Black Friday date expires. Better have the layout shift during to draw people's attention.

Reply:
Handling it in #44632